corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -45,8 +45,11 @@
       <select>
     <% } else if (input == "daterange") { %>
       <input type="text" class="daterange query-field form-control" value="<%- value %>">
+    <% } else if (input == "address") { %>
+      <div class="query-field" style="width: 100%, max-width: none" value="<%- value %>" id="<%- id %>_mapbox" data-address="<%- id %>">
```

Comment:
Could you add a new class instead of using the inline styles?

Reply:
Actually, this is not necessary, removing it.